test/timestamps.test.js

```diff
@@ -626,6 +626,20 @@ describe('timestamps', function() {
       assert.strictEqual(cat.updatedAt, old);
     });
 
+    it('can skip with `$timestamps(false)` (gh-12117)', async function() {
+      const cat = await Cat.findOne();
+      const old = cat.updatedAt;
+
+      await delay(10);
```

Comment:
is this 10ms? just asking to be sure that the test does not time-out

Reply:
Yeah ms